src/runtime/contrib/tensorrt/tensorrt_builder.h

```diff
@@ -0,0 +1,210 @@
+/* * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+/*!
+* \file runtime/contrib/tensorrt/tensorrt_builder.h
+* \brief Contains TensorRTBuilder class which can be used to convert a relay
+* program into a TRT engine which can be used for inference.
+*/
+
+#ifndef TVM_RUNTIME_CONTRIB_TENSORRT_TENSORRT_BUILDER_H_
+#define TVM_RUNTIME_CONTRIB_TENSORRT_TENSORRT_BUILDER_H_
+
+#include <tvm/relay/expr_functor.h>
+#include <tvm/relay/type.h>
+#include <tvm/runtime/ndarray.h>
+
+#include <string>
+#include <unordered_map>
+#include <vector>
+#include "NvInfer.h"
+
+#define TRT_VERSION_GE(major, minor, patch)                    \
+  ((NV_TENSORRT_MAJOR > major) ||                              \
+  (NV_TENSORRT_MAJOR == major && NV_TENSORRT_MINOR > minor) || \
+  (NV_TENSORRT_MAJOR == major && NV_TENSORRT_MINOR == minor && \
+  NV_TENSORRT_PATCH >= patch))
```

Comment:
Align

Reply:
The jenkins CI complains when this is alligned properly. I had to modify it like this to get it to pass.